rio_rgbify/mbtiler.py

```diff
@@ -249,10 +249,13 @@ def __init__(self, inpath, outpath, min_z, max_z, interval=1, base_val=0, boundi
 
         if not 'format' in kwargs:
             writer_func = _encode_as_png
+            self.format = 'png'
```

Comment:
@andrewharvey tests are failing because `.format` is a method reserved for strings, eg:
```python
'{}'.format(variable)
```
I think if we change this to `self._format` or `self.image_format` we should be fine, and tests will pass.